### DISCUSSION TITLE
Is there a way to change the Title of Landing/Login page?

### DISCUSSION THREAD

**Discussion by mateyelative** [modoboa/modoboa#3114] [Help]
Hey,
I am wondering if there is a way to change the Title of Landing/Login pag. 

I managed to change the logo, however  I am wondering if I can change also the Title of the page? Currently it is "Welcome to Modoboa".

Can someone direct me where I can find this parameter to change it.

Thank you a lot.
<img width="1508" alt="Screenshot 2023-11-07 at 15 58 15" src="https://github.com/modoboa/modoboa/assets/82472797/acdb31f3-0cd2-4ef0-b49c-4c4ae595453d">

**Comment by dominikusbrian** [comment]
![image](https://github.com/modoboa/modoboa/assets/60097047/f026ba96-aa12-43ef-a897-5f4599640db0)
I managed to change my website title (the one showing on browser) by modifying the index.html located here: /srv/modoboa/instance/frontend

**Reply by dominikusbrian** [reply]
![image](https://github.com/modoboa/modoboa/assets/60097047/43990b60-eea6-4e83-8174-c046e22ad47e)

**Reply by kalungia** [reply]
This did not work for me

**Reply by mateyelative** [reply]
Well, it does not work for me as well... Thank you @dominikusbrian.

**Reply by subznrl** [reply]
Please find the setting.py inside the /instance/instance and uncomment the custom logo section. Place the file named custom_logo.png on media folder /media/
The logo will be changed

**Comment by porrascarlos802018** [comment]
same here, edited , reloaded nginx same thing , welcome to modoboa...

**Comment by lane-ftw** [comment]
edit `/srv/modoboa/env/lib/python3.10/site-packages/modoboa/templates/registration/base.html`
Specifically 
`<title>{% trans "Welcome to Modoboa" %}</title>`
Line 8.
This is assuming you used the installer.

**Reply by subznrl** [reply]
Worked for me, Thank you
